### PR TITLE
Be able to build demo application on Jenkins with a build Docker image

### PR DESCRIPTION
**Pull request by sbrunner**

### CODE REVIEW

File: Dockerfile [hunk 1]
```diff
@@ -1,12 +1,76 @@
-FROM camptocamp/geomapfish_build_dev
+FROM camptocamp/geomapfish_build_dev:2,2
```
Comment (fredj) [inline Dockerfile]: is it really is comma between the `2` ?

Comment (sbrunner) [inline Dockerfile]: oups ...

Comment (sbrunner) [inline Dockerfile]: Fixed :-)

**Comment by sbrunner** [comment]
The Travis checks can't works while it's not merger, the reference check is https://travis-ci.org/camptocamp/c2cgeoportal/builds/263387976

**Comment by sbrunner** [comment]
Will also fir the 2.2 branch :-)

File: Dockerfile [hunk 2]
```diff
@@ -1,12 +1,76 @@
...
+    nomnom@1.8.1 \
+    openlayers@git://github.com/openlayers/openlayers#c611891 \
+    phantomjs-prebuilt@2.1.14 \
+    proj4@2.4.3 \
+    svg2ttf@4.1.0 \
+    temp@0.8.3 \
+    ttf2eot@2.0.0 \
+    ttf2woff@2.0.1 \
+    typeahead.js@0.11.1 \
+    walk@2.3.9
```
Comment (pvalsecc) [inline Dockerfile]: && rm -rf ~/.npm

File: Dockerfile [hunk 3]
```diff
@@ -1,12 +1,76 @@
...
+  for LANG in en fr de it en-ch fr-ch de-ch it-ch; \
+  do \
+    wget -O /opt/angular-locale/angular-locale_$LANG.js https://raw.githubusercontent.com/angular/angular.js/`grep ^angular.js= /usr/lib/node_modules/ngeo/github_versions | cut --delimiter = --fields 2 | tr --delete '\r\n'`/src/ngLocale/angular-locale_$LANG.js; \
+  done
+
+
+
+COPY . /opt/c2cgeoportal
+
+RUN pip install --disable-pip-version-check --editable /opt/c2cgeoportal
```
Comment (pvalsecc) [inline Dockerfile]: --no-cache-dir

File: Dockerfile [hunk 4]
```diff
@@ -1,12 +1,76 @@
-FROM camptocamp/geomapfish_build_dev
+FROM camptocamp/geomapfish_build_dev:2.2
 LABEL maintainer Camptocamp "info@camptocamp.com"
 
-COPY . /tmp/
+RUN \
+  echo "deb http://apt.dockerproject.org/repo debian-jessie main" >> /etc/apt/sources.list && \
+  apt-key adv --keyserver hkp://p80.pool.sks-keyservers.net:80 --recv-keys F76221572C52609D && \
+  apt-get update && \
+  apt-get install --assume-yes --no-install-recommends default-jre-headless docker-engine && \
```
Comment (pvalsecc) [inline Dockerfile]: default-jre-headless is already installed in the FROM

File: docker/build/Dockerfile [hunk 5]
```diff
@@ -15,10 +15,7 @@ COPY base-requirements.txt dev-requirements.txt fixversions.txt /tmp/
 
 RUN \
   cd /tmp && \
-  curl https://bootstrap.pypa.io/get-pip.py > get-pip.py && \
-  python get-pip.py `cat base-requirements.txt` && \
-  pip install --requirement dev-requirements.txt && \
-  apt-get clean && \
+  pip install --disable-pip-version-check --requirement dev-requirements.txt && \
```
Comment (pvalsecc) [inline docker/build/Dockerfile]: --no-cache-dir

**Pull request merged**